yb-voyager/cmd/config.go

```diff
@@ -253,6 +256,9 @@ func initConfig(cmd *cobra.Command) ([]ConfigFlagOverride, []EnvVarSetViaConfig,
 		return nil, nil, nil, fmt.Errorf("failed to bind environment variables to viper: %w", err)
 	}
 
+	// Set the in-memory config file for later use
+	InMemoryConfigFile = v.AllSettings()
```

Comment:
Minor aspect: 
The problem with saving the entire config file is that it opens up the possibility of someone using this data directly somewhere else. We don't want that because we want the flow to be viper -> cobra cli -> variables. (i.e. our main code always uses variables and not directly interact with cobra/viper). 

How about: 
- If the current command is export-data-from-source, only save the "import-data-to-source." keys in a specific variable `importDataToSourceConfigKeys` 
- if the current command  is import-data-to-target, only save the "export-data-from-target" keys in a specific variable `exportDataFromTargetConfigKeys`

Reply:
Given the recent discussion added code to read the config file dynamically before starting the command itself. 
Since the user might change the config file while the import data to target is running and we need the latest config file before we run the export data from target.